apps/projects/templates/meinberlin_projects/project_list.html

```diff
@@ -14,7 +14,8 @@
             {% if show %}
```

Comment:
if there are only projects the user is not allowed to view the alternative `{% empty %}` block won't be displayed. i am not sure how we could fix this. we would have to either filter the queryset in the backend or count the actually displayed projects